lib/parse.js

```diff
@@ -11,7 +11,7 @@ function create(matter) {
   var newline = '\n'
 
   frontmatter.displayName = name
-  frontmatter.onlyAtStart = true
+  frontmatter.onlyAtStart = matter.anywhere ? !matter.anywhere : true
```

Comment:
I think this would be nicer, I think this currently reads a bit confusing

```js
var anywhere = matter.anywhere
// ... 
onlyAtStart = typeof anywhere === 'boolean' ? !anywhere : true
```